ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -453,6 +453,12 @@ class SignalSpec: QuickSpec {
 				observer.sendNext(1)
 				expect(lastValue).to(equal("2"))
 			}
+
+			it("should not keep resulting signal alive indefinitely") {
+				weak var signal: Signal<AnyObject, NoError>? = Signal.never.map { $0 }
```

Comment:
@NachoSoto @andersio Yeah, the thing is, I don't know whether to test for one thing or for a completely opposite one. Why don't we try to deal with this simple case, unless anyone thinks that it really should leak.